packages/wrangler/src/__tests__/constellation.test.ts

```diff
@@ -0,0 +1,64 @@
+import { endEventLoop } from "./helpers/end-event-loop";
+import { mockConsoleMethods } from "./helpers/mock-console";
+import { runInTempDir } from "./helpers/run-in-tmp";
+import { runWrangler } from "./helpers/run-wrangler";
+
+describe("constellation", () => {
```

Comment:
This file should have tests for the command behaviour, rather than just snapshot tests

Reply:
It should. But all commands are basically a proxy to the API, via `fetchResult`. I looked for but couldn't find any support for mocking responses in `cfetch`.

If there is support for mocking API responses, can you please point me to it?

Reply:
The [publish](packages/wrangler/src/__tests__/publish.test.ts) tests have some API tests. We use MSW for API mocking. https://github.com/cloudflare/workers-sdk/blob/b5c3c897b01e7828254b2cac313cf2bd76592aba/packages/wrangler/src/__tests__/publish.test.ts#L7719 is a good example. @JacobMGEvans should be able to help with any specific questions—he's the MSW guru!

Reply:
Done.